psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -102,6 +103,11 @@ def __init__(self, alias, dtype, **kwargs):
                 if len(tuple_params) > 4:
                     raise Exception("Too many parameter in input tuple param.")
 
+        # DFT-NL dispersion. 
+        elif self.dtype in ["-NL"]:
+             self.disp_type = 'nl'
```

Comment:
If #986 gets merged before this, I've changed the suffixes to lowercase (-d2p4, -d3 etc.) to be consistent with the rest of the code. I guess the "-NL"'s scattered throughout the file should be lowercased too - I suppose you can fix that during rebase from master.